### DISCUSSION TITLE
LetsEncrypt SSL not renewing

### DISCUSSION THREAD

**Discussion by akchro** [vercel/community#4573] [Help]
### Summary

I originally hosted a website on DigitalOcean where I made a LetsEncrypt SSL for my domain. I later moved the website to Vercel and I heard that Vercel automatically renews SLL domains. My SSL certification is going to expire soon and Vercel hasn't renewed it yet. How can I have Vercel renew my SSL certificate?

Note: I have ``0 issue "letsencrypt.org"`` as one of my CAA records.

### Steps to Reproduce

1. Host a domain on a different service or self host and setup LetsEncrypt for that domain.
2. Move domain to Vercel.

**Comment by amyegan** [comment]
If a domain has a valid configuration, then the SSL certificate will automatically renew when needed. If the domain is misconfigured, then then certificate may not be renewed. Please visit the following pages for more info:

- [Working with SSL Certificates](https://vercel.com/docs/projects/domains/working-with-ssl#)
- [When is the SSL Certificate on my Vercel Domain renewed?](https://vercel.com/guides/renewal-of-ssl-certificates-with-a-vercel-domain)
- [Why is my domain not automatically generating an SSL certificate?](https://vercel.com/guides/domain-not-generating-ssl-certificate)
- [Common SSL certificate issues](https://vercel.com/docs/projects/domains/troubleshooting#common-ssl-certificate-issues)
